File: src/webapp/src/app/model/college.ts
export class College {

  id: number;
  siteName: string;
  displayName: string;
  conference: string;
  division: string;

  constructor(id: number, siteName: string, displayName: string, conference: string, division: string) {
    this.id = id;
    this.siteName = siteName;
    this.displayName = displayName;
    this.conference = conference;
    this.division = division;
  }

}

File: src/main/java/com/scurtis/recruits/dto/College.java
package com.scurtis.recruits.dto;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 * Author: Steve Curtis
 * Date: Nov 26, 2020
 **/

@Data
@Entity
@Table(name = "colleges")
public class College {

    @Id
    private Integer id;
    private String siteName;
    private String displayName;
    private String conference;
    private String division;

}
